### ISSUE TITLE
Use Scrcpy as Second Display?

### ISSUE THREAD

**Issue by biomagnet** [Genymobile/scrcpy#4708]
- [x] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.  
(I actually did [find this](https://github.com/Genymobile/scrcpy/issues/886) from 2019 but I'm not sure if it's all the same thing or not. Please excuse my lack of knowledge.)

**Is your feature request related to a problem? Please describe.**
Trying to set up a second display on my device without the appropriate cables.

**Describe the solution you'd like**
If possible, I'd like an option where connecting to Scrcpy makes the new window it creates to act as a second monitor/display entirely. Ideally it would then be able to do [silly things like this](https://www.reddit.com/r/retroid/comments/1ay1yco/not_a_bad_way_to_play_ds3ds_games/) without needing to use up the entirely of a monitor (useful for keeping other things up in the background, such as guides or chat programs if you only have one display to use in the first place).

**Describe alternatives you've considered**
Buying a USB-C to HDMI cable, ig.

**Additional context**
I checked [video.md](https://github.com/Genymobile/scrcpy/blob/master/doc/video.md) and I see *something* about "display" but I'm sadly not very good at understanding commands like those directly. As far as I can see, it only mirrors things that are already attached rather than being able to act as something new itself? If such a thing is possible, I'd kindly ask for an explanation of how to do it.

**Comment by rom1v** [comment]
It's currently not supported.

See threads #397 and #1887.

**Comment by biomagnet** [comment]
Hopefully this is not considered necroposting, but would #5370 be what I'm looking for if/when it is finished and added as a feature? I didn't want to start a new issue to ask.

**Comment by rom1v** [comment]
> would https://github.com/Genymobile/scrcpy/pull/5370 be what I'm looking for if/when it is finished

The current state is what the feature will look like. Currently, only an option to start an app is missing (I'm working on it).

**Comment by biomagnet** [comment]
That's excellent! I don't know how to build it early myself or anything, so I'll simply cheer you on to get the last of the bugs worked out and wait to see "5370" in the notes of a new release. Thanks very much for the confirmation.

**Comment by rom1v** [comment]
> I don't know how to build it early myself or anything

There's a binary in #5370.

**Comment by biomagnet** [comment]
Oh, I found it! It does definitely make a second android display/screen... but whatever it's making doesn't read as "an external display" for the app I'm trying to use (DraStic) to split the DS outputs. That's not on you, though, so more than anything I'm happy this function works.

**Comment by qiine** [comment]
I might have misunderstood the discussion, but the feature that is wanted would be more like allowing you to use a tablet for example, as a second monitor for your windows/linux machine ? 

Essentially turning your Android device into an usb  power computer screen.

**Comment by biomagnet** [comment]
No, you have it backwards. Using the computer monitor (and/or application window) as a second monitor for the Android device. This is useful for certain types of gaming emulation, such as DS, 3DS and Wii U. Especially without cables or any other barrier of entry for doing so, such as ScrCpy allows.

**Comment by M0hit-dev** [comment]
I think it's now possible with:
`scrcpy --new-display`

What i use:
**Developer Options > Force desktop mode**
`scrcpy --fullscreen --new-display=1366x769/240 --keyboard=uhid --mouse=uhid --max-fps=60 `

**Comment by FiqihHakimi** [comment]
wait. @M0hit-dev how to apply that exactly ? becouse i dont event know about coding or programing

**Comment by rom1v** [comment]
@M0hit-dev Read https://github.com/Genymobile/scrcpy/blob/master/doc/windows.md#run
